snapcraft.yaml

```diff
@@ -788,7 +788,7 @@ parts:
   gtkmm:
     after: [ atkmm, meson-deps ]
     source: https://gitlab.gnome.org/GNOME/gtkmm.git
-    source-tag: '4.16.0'
+    source-tag: '4.17.0'
```

Comment:
This is a beta release. 4.18.0 is out now as the stable release. I suggest updating to that before releasing this to stable.

Reply:
Then we must add an `ignore-odd-minor` statement to avoid using develop versions. I'll do it right now too. Do you know of other parts that still use the odd->development even->stable numbering system that don't have it?

Reply:
@jbicha as our resident GNOME Release Manager might

Reply:
Done too.